packages/fxa-auth-server/lib/senders/emails/templates/postVerifySecondary/index.txt

```diff
@@ -1,6 +1,6 @@
 postVerifySecondary-title = "Secondary email added"
 
-postVerifySecondary-description = "You have successfully added <%- secondaryEmail %> as a secondary email from your Firefox account. Security notifications and sign-in confirmations will now be delivered to both email addresses."
+postVerifySecondary-description = "You have successfully verified <%- secondaryEmail %> as a secondary email from your Firefox account. Security notifications and sign-in confirmations will now be delivered to both email addresses."
```

Comment:
Unfortunately we have to update the ftl ID any time we update a string because the ID is now forever tied to this string in Pontoon. You can make this `postVerifySecondary-description-2` here and in the template.